packages/Progress/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@tds/community-progress",
+  "version": "0.0.0",
```

Comment:
Please bump version to `0.1.0` or higher to avoid an issue with the automated package management via `Lerna`.

Reply:
done.